### DISCUSSION TITLE
How can I edit the RSS Feed of the Main Stream?

### DISCUSSION THREAD

**Discussion by CharlesMingus19** [FreshRSS/FreshRSS#5644] [Q&A]
Hello,

I wanted to add a custom string in the description of each item in my RSS feed of the main stream.
I was able to do it with the Custom JS extension but this only edits the feed when viewing through the FreshRSS platform.
It does not edit the RSS feed of the main stream: (example.com/p/i/?a=rss)
How can I add this string so it also shows up in the rss feed?

**Comment by Frenzie** [comment]
In an extension you can change various attributes, see for example https://github.com/FreshRSS/Extensions/blob/f6d2b21d68118c920c4c4c06bfc05307271218e0/xExtension-ImageProxy/extension.php#L131-L137

Although depending on what you're looking for perhaps it'd make more sense to do the same thing prior to database entry.

**Reply by CharlesMingus19** [reply]
Ah I don't think I can change the attributes of an extension as I am using a cloud provider.

Specifically, I wanted the Feed Title to appear in each item's description of the RSS feed:
![image](https://github.com/FreshRSS/FreshRSS/assets/144540060/5a488ee5-03ba-45af-9583-6acf3b8927da)

**Reply by Frenzie** [reply]
In that case I doubt what you're looking for is possible.
